### ISSUE TITLE
Support multi upload of images?

### ISSUE THREAD

**Issue by steinmb** [elmcip/elmcip#380]
During the Montreal ELO ELMCIP workshop session there was a request on supporting multi image upload. Screenshots etc. Save to creating new records.

**Comment by elmcip** [comment]
We can already upload multiple images. The issue is doing them all at the same time?

**Comment by steinmb** [comment]
Yes. Not having to add them one by one. Useful though perhaps not the most important feature request raised.
